docs/GuildConfigSchema.md

```diff
@@ -0,0 +1,58 @@
+# SudoBot Configuration
+
+SudoBot's behavior is controlled through a comprehensive JSON schema. This README provides an overview of the Guild Configuration Schema, allowing you to tailor the bot's behavior to suit your server's needs. Refer to [GuildConfigSchema.ts](GuildConfigSchema.ts) for the complete schema.
+
+## Guild Configuration Schema
+
+The Guild Configuration Schema encompasses various settings for SudoBot. Below are detailed explanations for key sections of the schema:
+
+### Prefix and Debug Mode
+
+- **`prefix`**: Defines the command prefix for SudoBot. By default, commands are triggered with a prefix of `-`. You can customize this to align with your server's preferences.
+
+- **`debug_mode`**: When set to `true`, enables debug mode, providing additional information useful for developers. Typically, debug mode is disabled in production environments.
+
+### Commands Configuration
+
+- **`commands`**: Configuration related to bot commands.
+
+  - **`mention_prefix`**: If set to `true`, allows commands to be triggered with mentions (e.g., `@SudoBot help`). If set to `false`, only the command prefix triggers commands.
+
+  - **`bean_safe`**, **`shot_safe`**, **`fakeban_safe`**: Arrays of Snowflake IDs for safe actions. Users with these IDs are exempt from certain actions.
+
+  - **`echo_mentions`**: Toggles echoing of mentions in command responses. If set to `true`, mentions in command responses are echoed; otherwise, they are removed.
```

Comment:
Not really, the `echo_mentions` option tells the bot whether to also ping roles when someone uses the echo comamnd.